File: frontEnd/src/app/types/applicants.ts
export class Applicants {
  public applicantID: number;
  public lastName: string;
  public firstName: string;
  public city: string;
  public telephone: string;
  public email: string;
  public password: string;

  constructor(
    applicationID: number,
    lastName: string,
    firstName: string,
    city: string,
    telephone: string,
    email: string,
    password: string,
  ) {
    this.applicantID = applicationID;
    this.lastName = lastName;
    this.firstName = firstName;
    this.city = city;
    this.telephone = telephone;
    this.email = email;
    this.password = password;
  }
}

File: BackendServer/BackendService/Model/Applicants.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BackendService.Model
{
    [Table("applicants")]
    public class Applicants
    {
        [Key]
        public int ApplicantID { get; set; }
        public string LastName { get; set; }
        public string FirstName { get; set; }
        public string Password { get; set; }
        public string City { get; set; }
        public string Postcode { get; set; }
        public string Email { get; set; }
        public string Telephone { get; set; }
    }
}
